script/foreman-puma-status

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+puma-status /run/foreman/puma.state
```

Comment:
Does this work on a Mac?

Reply:
I doubt it does, but running `puma-status ~foreman/tmp/puma.state` probably does. Unless you use a different user in which case it doesn't really scale. You could accept an argument but then you may also need to use bundle exec. I think this file is more intended for production.

Reply:
Is the foreman server expected to be running on a Mac OS?

Reply:
Just the dev environment. I'm good with merging this, we can just add a comment `Intended for a production environment`.

Reply:
@timogoebel ty for the clarification. Added a comment in the script now.